Makefile

```diff
@@ -30,6 +30,10 @@ lint-jsonnet: $(JSONNET_LINT) jsonnet-vendor
 		-o -name '*.jsonnet' -print \
 	| xargs -n 1 -- $(JSONNET_LINT) -J $(JSONNET_VENDOR)
 
+.PHONY: lint-shell
+lint-shell:
+	find -name "*.sh" -print0 | xargs --null shellcheck
```

Comment:
Shouldn't we use $(SHELLCHECK) instead of `shellcheck` to make use of the shellcheck installed to tmp/bin?